pipelines/_ingredients/LSSTComCam/DRP.yaml

```diff
@@ -11,6 +13,16 @@ imports:
   - $ANALYSIS_TOOLS_DIR/pipelines/wholeSkyCore.yaml
 tasks:
   isr: lsst.ip.isr.IsrTaskLSST
+  characterizeImage: lsst.pipe.tasks.characterizeImage.CharacterizeImageTask
```

Comment:
make a note here why you are not using the thing someone would see everywhere else (calibrateImage)